android/src/com/androidAuto/HelloWorldScreen.java

```diff
@@ -0,0 +1,211 @@
+package com.androidAuto;
+
+import android.graphics.Canvas;
+import android.graphics.Rect;
+import android.util.Log;
+import android.view.Surface;
+import android.opengl.GLUtils;
+import com.mapswithme.util.log.Logger;
+import androidx.annotation.NonNull;
+import androidx.car.app.AppManager;
+import androidx.car.app.CarContext;
+import androidx.car.app.Screen;
+import androidx.car.app.SurfaceCallback;
+import androidx.car.app.SurfaceContainer;
+import androidx.car.app.model.Action;
+import androidx.car.app.model.ActionStrip;
+import androidx.car.app.model.Template;
+import androidx.car.app.navigation.NavigationManager;
+import androidx.car.app.navigation.model.NavigationTemplate;
+import com.mapswithme.maps.BuildConfig;
+import com.mapswithme.maps.MapFragment;
+import com.mapswithme.maps.MwmApplication;
+import com.mapswithme.maps.location.LocationHelper;
+import javax.microedition.khronos.egl.EGL10;
+import javax.microedition.khronos.egl.EGLConfig;
+import javax.microedition.khronos.egl.EGLContext;
+import javax.microedition.khronos.egl.EGLDisplay;
+import javax.microedition.khronos.egl.EGLSurface;
+
+import java.io.IOException;
+
+public class HelloWorldScreen extends Screen implements SurfaceCallback
+{
+  private static final String TAG = "error";
+  private SurfaceCallback mSurfaceCallback;
+
+  public HelloWorldScreen(@NonNull CarContext carContext)
+  {
+    super(carContext);
+
+    carContext.getCarService(AppManager.class).setSurfaceCallback(this);
+  }
+
+  @NonNull
+  @Override
+  public Template onGetTemplate()
+  {
+    Log.e("Show Up", "Shows Up");
```

Comment:
Istead of `Log.e(...)` you better use `Log.w(...)` for warnings and `Log.i(...)` for information output and `Log.d(...)` for debug.